### PR TITLE
I use Arch btw

### PR DESCRIPTION
**Pull request by TornaxO7**
I use Arch btw.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -117,7 +117,8 @@
     "arch?",
     "Habe ich Arch gehört?",
     "Schonmal Arch installiert?",
-    "Arch for the win!"
+    "Arch for the win!",
+    "https://www.youtube.com/watch?v=eb0hUjewRgE",
```
Comment (MultisampledNight) [inline main.py:121]: Please include a loud sound warning :D

**Review by MultisampledNight — CHANGES_REQUESTED** [review]
Except adding a note for the large sound would be worth it, seems good so far. Whether that stuff isn't too... (you know what I want to say about the second scene in the video) is up to you.

File: main.py (hunk 1 above)
Comment (TornaxO7) [reply main.py:121]: what do you mean with that?

**Comment by TornaxO7** [comment]
Done

**Review by MultisampledNight — APPROVED** [review]
Looks good to me. Thanks! Since @m-braunschweig seems to be unresponsive, I think we can merge this now.

**Pull request merged**